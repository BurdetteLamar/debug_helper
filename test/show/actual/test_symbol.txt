---
Symbol (size=11 name=test_symbol): :lorem_ipsum
