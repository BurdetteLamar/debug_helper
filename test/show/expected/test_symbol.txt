---
Symbol (message='test_symbol' size=11 encoding=US-ASCII): :lorem_ipsum
